app/Model/Event.php

```diff
@@ -2346,6 +2346,8 @@ public function fetchEvent($user, $options = array(), $useCache = false)
                 $results[$k] = $this->__mergeExtensions($user, $result['Event']['uuid'], $result);
             }
         }
+        // Clear tags cache after fetching all events, because another fetch can use different tag conditions.
+        $this->assetCache['tags'] = [];
```

Comment:
Is that something we really want? The idea was to persist the assetCache between the execution of individual events for restSearch for example (which uses a lot of chunked fetchEvent() calls).

Reply:
It was just temporary fix, changed now. But I still need test that.